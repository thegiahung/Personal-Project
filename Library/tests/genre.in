List genres
Genre comedy
add collection books.csv
list genres
genre War
genre Comedy
genre haha
genre Hello World
exit
