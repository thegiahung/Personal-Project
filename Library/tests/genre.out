user: No books in system.

user: No books in system.

user: 17 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Historical Fiction
Horror
Poetry
War

user: Biggles in the Battle (W.E. Johns)

user: The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)

user: No books with genre haha.

user: No books with genre Hello World.

user: Ending Library process.
